src/pages/documentation/adobe-journey-optimizer/tutorials/handle-clicks.md

```diff
@@ -75,6 +75,14 @@ The `link` parameter must be URL encoded. If you are setting this value in the A
 
 ### Open a deeplink
 
+<InlineAlert variant="help" slots="text"/>
+
+Android deeplinks require intent filters to be added in the app manifest to correctly handle deeplink URL's. Please see the Android developer documentation for more information: https://developer.android.com/training/app-links/deep-linking
```

Comment:
URLs should just be URLs, not "URL's".

Also `Please see the [Android developer documentation](https://developer.android.com/training/app-links/deep-linking) for more information.`